test/remote/gateways/remote_orbital_test.rb

```diff
@@ -442,6 +599,23 @@ def test_refund
     assert_success refund
   end
 
+  # Successful Refund (A) of amount on purchase through Electronic Check
+  def test_successful_refund_with_echeck
+    assert response = @gateway.purchase(@amount, @echeck, @options)
+    assert_success response
+    assert response.authorization
+    assert refund = @gateway.refund(@amount, response.authorization, @options)
+    assert_success refund
+  end
+
+  # Failed refund due to invalid Transaction provided
+  def test_failed_refund_with_echeck_due_to_invalid_authorization
+    assert refund = @gateway.refund(@amount, '123;123', @options)
+    assert_failure refund
+    assert_equal 'The LIDM you supplied (3F3F3F) does not match with any existing transaction', refund.message
```

Comment:
Should this be 123;123 instead of 3F3F3F ?

Reply:
Whatever value we provide in the authorization string, server produces response that has 3F3F3F and please note that 123 doesn't mean something special but it just an invalid number passed to check the failure case.

Reply:
Okay thanks for the information!